ui/entity/EntityData.tsx

```diff
@@ -72,6 +92,16 @@ const EntityData = ({ entityQuery }: Props) => {
           </ItemValue>
         </>
       ) }
+
+      <ItemDivider/>
+      <ItemLabel hint="Entities related to this entity">Related Entities</ItemLabel>
+      <ItemValue>
+        <Skeleton loading={ isLoading }>
+          <Link href={ `/entity?${ annotationsQuery }` }>
```

Comment:
Use `route({ pathname: '/entity', query: {...})` helper for building type safe URL.